src/components/RadioGroupField/RadioGroupField.tsx

```diff
@@ -77,22 +77,22 @@ export const RadioGroupField = withStyles(styles, {
           onChange={onChange}
         >
           {choices.length > 0 ? (
-            choices.map(choice => {
-              return (
-                <FormControlLabel
-                  value={choice.value}
-                  className={classes.radioLabel}
-                  control={<Radio color="primary" />}
-                  label={
-                    <>
-                      {choice.label}
-                      <span>{choice.secondLabel}</span>
-                    </>
-                  }
-                  key={choice.value}
-                />
-              );
-            })
+            choices.map(choice => (
+              <FormControlLabel
+                value={choice.value}
+                className={classes.radioLabel}
+                control={<Radio color="primary" />}
+                label={
+                  <>
+                    {choice.label}
+                    <span className={classes.secondLabel}>
```

Comment:
I think there was a misunderstanding - I asked that we drop `secondLabel` instead of naming its class